src/screens/ChangeWalletSheet.js

```diff
@@ -264,11 +265,7 @@ export default function ChangeWalletSheet() {
                     await dispatch(walletsSetSelected(updatedWallet));
                   }
 
-                  updateWebProfile(
-                    address,
-                    args.name,
-                    colors.avatarBackgrounds[args.color]
-                  );
+                  updateWebProfile(address, args.color, args.emoji);
```

Comment:
should we check for watched wallet as you did before?

Reply:
actually ended up removing this, it seems that the ability to change the avatar while editing the wallet no longer exists